{
    "title" : "Could not establish trust relationship for SSL_TLS after a app pool recycle",
    "uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US",
    "printableUri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US",
    "clickUri" : "https://na61.salesforce.com/kA132000000L6sl",
    "uniqueId" : "42.13819$http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US",
    "excerpt" : "Using Coveo for Sitecore 3.0 version after March 2016, you get this error message in the diagnostic page ... The remote server returned an error: (500) Internal Server Error ... Could not establish...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1397,
    "percentScore" : 84.103615,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "Could not establish trust relationship for SSL_TLS after a app pool recycle",
      "sysurihash" : "myyuJV81HE6nGFñ8",
      "urihash" : "myyuJV81HE6nGFñ8",
      "parents" : "<?xml version=\"1.0\" encoding=\"utf-16\"?><parents><parent name=\"Organization\" uri=\"https://na61.salesforce.com/home/home.jsp\" /><parent name=\"KB_Article__ka\" uri=\"http://www.salesforce.com/org:organization/articletype:KB_Article\" /><parent name=\"Could not establish trust relationship for SSL_TLS after a app pool recycle\" uri=\"https://na61.salesforce.com/kA132000000L6sl\" /></parents>",
      "sfdcarticletype" : "Problem/Fix",
      "sfresolutionc" : "Add the initializeOnAdd node and set it to true inside the defaultIndexConfiguration node.",
      "sysuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US",
      "sfreleasedatec" : "Jun 2016",
      "sysprintableuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US",
      "sflastpublisheddate" : 1476217823000,
      "systransactionid" : 161138,
      "sfurlname" : "1801",
      "sfsystemmodstamp" : 1517572013000,
      "sfisvisibleinapp" : "true",
      "sfcoveoversionc" : "Coveo for Sitecore 3.0",
      "sysconcepts" : "server ; initializeOnAdd ; defaultIndexConfiguration type ; single item ; App Pool ; trust relationship ; Using Coveo ; myCESServer ; Environment",
      "sfkbversionnumber" : "1",
      "concepts" : "server ; initializeOnAdd ; defaultIndexConfiguration type ; single item ; App Pool ; trust relationship ; Using Coveo ; myCESServer ; Environment",
      "printableuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US",
      "documenttype" : "article",
      "sfkbarticlekavid" : "ka132000000LGOlAAO",
      "sysindexeddate" : 1518187508000,
      "sfknowledgearticleid" : "kA132000000L6slCAC",
      "sfcreatedbyid" : "00560000003q7JgAAI",
      "sfconnectorc" : "N/A",
      "permanentid" : "c63324738b8e039143b174f85a0596209ed237cc365a7e4385ce7cf2c8af",
      "syslanguage" : [ "English" ],
      "transactionid" : 161138,
      "sfisvisibleinprm" : "true",
      "title" : "Could not establish trust relationship for SSL_TLS after a app pool recycle",
      "sfversionnumber" : 1,
      "sfarticlenumber" : "000001801",
      "date" : 1517572013000,
      "objecttype" : "KB Article",
      "allmetadatavalues" : "{\"kav_lastmodifiedby.attribute.type\": [\"User\"], \"coveo_short_id\": [\"kA132000000L6sl\"], \"sflastmodifiedbyname\": [\"Frederic Cote\"], \"kav_coveo_version__c__html_stripped\": [\"Coveo for Sitecore 3.0\"], \"fileextension\": [\"\"], \"sfdcexpandedproduct\": [\"All;All|Sitecore\"], \"kav_createdby.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000003q7JgAAI\"], \"sflanguage\": [\"en_US\"], \"kav_createdby.attribute.type\": [\"User\"], \"kav_createdby.id\": [\"00560000003q7JgAAI\"], \"kav_language\": [\"en_US\"], \"sfsitecoreversionc\": [\"N/A\"], \"sourcetype\": [\"Salesforce\"], \"title\": [\"Could not establish trust relationship for SSL_TLS after a app pool recycle\"], \"source\": [\"Salesforce Public Items\"], \"kav_owner.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000003q7JgAAI\"], \"parents\": [\"<?xml version=\\\"1.0\\\" encoding=\\\"utf-16\\\"?><parents><parent name=\\\"Organization\\\" uri=\\\"https://na61.salesforce.com/home/home.jsp\\\" /><parent name=\\\"KB_Article__ka\\\" uri=\\\"http://www.salesforce.com/org:organization/articletype:KB_Article\\\" /><parent name=\\\"Could not establish trust relationship for SSL_TLS after a app pool recycle\\\" uri=\\\"https://na61.salesforce.com/kA132000000L6sl\\\" /></parents>\"], \"sfisvisibleinprm\": [true], \"kav_title\": [\"Could not establish trust relationship for SSL/TLS after a app pool recycle\"], \"kav_isvisibleincsp\": [true], \"hashtmlversion\": [true], \"kav_versionnumber\": [1], \"sfcreatedbyname\": [\"Frederic Cote\"], \"sflastpublisheddate\": [1476217823], \"objecttypelabel\": [\"KB Article\"], \"kav_cause__c__html_stripped\": [\"This node in\\u00a0Coveo.SearchProviger.config \\n \\u00a0\\n \\n<defaultIndexConfiguration type= ... > \\n \\n Is missing this children node: \\n \\u00a0\\n \\n<initializeOnAdd>true</initializeOnAdd> \\n \\n \\u00a0\"], \"originalhtmlcharset\": [1200], \"coveo_connector_hasbinarydata\": [false], \"kav_lastpublisheddate\": [1476217823], \"sfsystemmodstamp\": [1517572013], \"ka_attribute.type\": [\"KB_Article__ka\"], \"sfdcproduct\": [\"Sitecore\"], \"sfvendorversionc\": [\"Sitecore 7.2.160123\"], \"kav_resolution__c__html_stripped\": [\"Add the\\u00a0initializeOnAdd node and set it to true inside the\\u00a0defaultIndexConfiguration node.\"], \"clickableuri\": [\"https://na61.salesforce.com/kA132000000L6sl\"], \"sftitle\": [\"Could not establish trust relationship for SSL/TLS after a app pool recycle\"], \"kav_owner.id\": [\"00560000003q7JgAAI\"], \"sfbuildc\": [\"1312\"], \"sfisvisibleinapp\": [true], \"ka_attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__ka/kA132000000L6slCAC\"], \"sfdcarticletype\": [\"Problem/Fix\"], \"kav_cause__c\": [\"This node in\\u00a0Coveo.SearchProviger.config<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\n&lt;defaultIndexConfiguration type= ... &gt;</pre>\\n<br>Is missing this children node:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\n&lt;initializeOnAdd&gt;true&lt;/initializeOnAdd&gt;</pre>\\n<br>\\u00a0\"], \"sfislatestversion\": [true], \"kav_release_date__c\": [\"Jun 2016\"], \"contenttype\": [\"binarydata\"], \"sfurlname\": [\"1801\"], \"ka_id\": [\"kA132000000L6slCAC\"], \"kav_datacategoryselections.datacategoryname\": [\"Configuration;Sitecore\"], \"coveo_url\": [\"https://na61.salesforce.com/kA132000000L6sl\"], \"kav_urlname\": [\"1801\"], \"sfreleasedatec\": [\"Jun 2016\"], \"kav_dc_expanded_product\": [\"All;All|Sitecore\"], \"foldingcollection\": [\"c15fc8662af4c538b7c54097636f2c79ba4fddddf5af567acc24afede7ed\"], \"body\": [\"<html>   <body>     <div style=\\\"background-color: #fff;border: 2px solid #BBC5CC;border-radius: 4px;float: left;font-family: lato;margin-left: 15px;margin-right: 15px;margin-bottom: 4%;padding: 10px;width: 60%;\\\">       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Description         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> Using Coveo for Sitecore 3.0 version after March 2016, you get this error message in the diagnostic page<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nThe remote server returned an error: (500) Internal Server Error\\u00a0\\nCould not establish trust relationship for the SSL/TLS secure channel with authority &#39;myCESServer:52810&#39;</pre>\\nIn addition, while this message is visible, no search can be performed<br><br>Also, this message occurs after each App Pool Recycle. Re indexing a single item solves the problem until the next recycle.</div>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Environment         </div>       </div>       <br/>       <table>         <tr>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Product</td>           <td></td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Sitecore Version</td>           <td>N/A</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Connector</td>           <td>N/A</td>         </tr>         <tr>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Published</td>           <td>1476217823</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Version</td>           <td></td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Build</td>           <td>1312</td>         </tr>       </table>       <br/>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Cause         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> This node in\\u00a0Coveo.SearchProviger.config<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\n&lt;defaultIndexConfiguration type= ... &gt;</pre>\\n<br>Is missing this children node:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\n&lt;initializeOnAdd&gt;true&lt;/initializeOnAdd&gt;</pre>\\n<br>\\u00a0</div>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Resolution         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> Add the\\u00a0initializeOnAdd node and set it to true inside the\\u00a0defaultIndexConfiguration node.</div>     </div>   </body> </html>\"], \"sfpublishstatus\": [\"Online\"], \"sfcreatedbyid\": [\"00560000003q7JgAAI\"], \"kav_description__c__html_stripped\": [\"Using Coveo for Sitecore 3.0 version after March 2016, you get this error message in the diagnostic page \\n \\u00a0\\n \\nThe remote server returned an error: (500) Internal Server Error\\u00a0\\nCould not establish trust relationship for the SSL/TLS secure channel with authority 'myCESServer:52810' \\nIn addition, while this message is visible, no search can be performed \\n \\n Also, this message occurs after each App Pool Recycle. Re indexing a single item solves the problem until the next recycle.\"], \"kav_islatestversion\": [true], \"kav_knowledgearticleid\": [\"kA132000000L6slCAC\"], \"sfknowledgearticleid\": [\"kA132000000L6slCAC\"], \"sfownername\": [\"Frederic Cote\"], \"kav_datacategoryselections.attribute.type\": [\"KB_Article__DataCategorySelection;KB_Article__DataCategorySelection\"], \"detectedtitle\": [\"The remote server returned an error: (500) Internal Server Error\"], \"kav_createddate\": [1476217361], \"kav_build__c\": [\"1312\"], \"language\": [\"English\"], \"coveo_foldingchild\": [\"5428021d628a3a6754a3d43ff2322323eb0ca24d8eeae772e728bff1ff18\"], \"sfownerid\": [\"00560000003q7JgAAI\"], \"kav_createdby.coveo_record_size\": [2], \"kav_vendor_version__c\": [\"Sitecore 7.2.160123\"], \"kav_systemmodstamp\": [1517572013], \"objecttype\": [\"KB Article\"], \"connectortype\": [\"Salesforce2\"], \"kav_coveo_record_size\": [4], \"originaluri\": [\"http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US\"], \"sfisvisibleincsp\": [true], \"convertertype\": [\"TYPE_HTML\"], \"filetype\": [\"SalesforceItem\"], \"kav_owner.name\": [\"Frederic Cote\"], \"sfcoveoversionc\": [\"Coveo for Sitecore 3.0\"], \"sfdcexpandedarticletype\": [\"All;All|Problem/Fix\"], \"objecttypelabelplural\": [\"KB Articles\"], \"sfcurrencyisocode\": [\"CAD\"], \"sfisdeleted\": [false], \"extractedsize\": [1748], \"size\": [3962], \"kav_publishstatus\": [\"Online\"], \"kav_connector__c\": [\"N/A\"], \"generateexcerpt\": [true], \"sfdescriptionc\": [\"Using Coveo for Sitecore 3.0 version after March 2016, you get this error message in the diagnostic page<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nThe remote server returned an error: (500) Internal Server Error\\u00a0\\nCould not establish trust relationship for the SSL/TLS secure channel with authority &#39;myCESServer:52810&#39;</pre>\\nIn addition, while this message is visible, no search can be performed<br><br>Also, this message occurs after each App Pool Recycle. Re indexing a single item solves the problem until the next recycle.\"], \"sfarticletype\": [\"KB_Article__kav\"], \"kav_articletype\": [\"KB_Article__kav\"], \"kav_lastmodifieddate\": [1476217823], \"documenttype\": [\"article\"], \"sfkbarticlekavid\": [\"ka132000000LGOlAAO\"], \"kav_currencyisocode\": [\"CAD\"], \"foldingparent\": [\"5428021d628a3a6754a3d43ff2322323eb0ca24d8eeae772e728bff1ff18\"], \"mappingtype\": [\"KB_Article\"], \"kav_isvisibleinpkb\": [true], \"kav_sourceid\": [\"50032000015dbB4AAI\"], \"sfkbversionnumber\": [1], \"kav_isvisibleinapp\": [true], \"kav_resolution__c\": [\"Add the\\u00a0initializeOnAdd node and set it to true inside the\\u00a0defaultIndexConfiguration node.\"], \"coveo_objecttype\": [\"KB_Article\"], \"printableuri\": [\"http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US\"], \"kav_datacategoryselections.attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__DataCategorySelection/02o32000000pjvaAAA;/services/data/v34.0/sobjects/KB_Article__DataCategorySelection/02o32000000pjvZAAQ\"], \"kav_id\": [\"ka132000000LGOlAAO\"], \"sfresolutionc\": [\"Add the\\u00a0initializeOnAdd node and set it to true inside the\\u00a0defaultIndexConfiguration node.\"], \"foldingchild\": [\"5428021d628a3a6754a3d43ff2322323eb0ca24d8eeae772e728bff1ff18\"], \"sfcreateddate\": [1476217361], \"kav_sitecore_version__c\": [\"N/A\"], \"kav_owner.coveo_record_size\": [2], \"kav_coveo_effective_modstamp\": [1517572013], \"kav_summary__html_stripped\": [\"\"], \"kav_dc_article_type\": [\"Problem/Fix\"], \"sfconnectorc\": [\"N/A\"], \"kav_dc_product\": [\"Sitecore\"], \"ka_coveo_record_size\": [2], \"kav_dc_expanded_article_type\": [\"All;All|Problem/Fix\"], \"kav_datacategoryselections.coveo_record_size\": [\"2;2\"], \"concepts\": [\"server ; initializeOnAdd ; defaultIndexConfiguration type ; single item ; App Pool ; trust relationship ; Using Coveo ; myCESServer ; Environment\"], \"sfkbid\": [\"kA132000000L6slCAC\"], \"kav_firstpublisheddate\": [1476217823], \"coveo_organization_id\": [\"00D3000000007r2\"], \"coveo_foldingcollection\": [\"c15fc8662af4c538b7c54097636f2c79ba4fddddf5af567acc24afede7ed\"], \"sffirstpublisheddate\": [1476217823], \"sflastmodifieddate\": [1476217823], \"kav_coveo_version__c\": [\"Coveo for Sitecore 3.0\"], \"kav_vendor_version__c__html_stripped\": [\"Sitecore 7.2.160123\"], \"detectedlanguage\": [1], \"kav_isvisibleinprm\": [true], \"kav_owner.attribute.type\": [\"Name\"], \"kav_lastmodifiedby.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000003q7JgAAI\"], \"sitename\": [\"Community\"], \"kav_createdby.name\": [\"Frederic Cote\"], \"sfarticlenumber\": [\"000001801\"], \"kav_lastmodifiedby.coveo_record_size\": [2], \"sfisvisibleinpkb\": [true], \"kav_datacategoryselections.id\": [\"02o32000000pjvaAAA;02o32000000pjvZAAQ\"], \"kav_description__c\": [\"Using Coveo for Sitecore 3.0 version after March 2016, you get this error message in the diagnostic page<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nThe remote server returned an error: (500) Internal Server Error\\u00a0\\nCould not establish trust relationship for the SSL/TLS secure channel with authority &#39;myCESServer:52810&#39;</pre>\\nIn addition, while this message is visible, no search can be performed<br><br>Also, this message occurs after each App Pool Recycle. Re indexing a single item solves the problem until the next recycle.\"], \"kav_attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__kav/ka132000000LGOlAAO\"], \"kav_articlenumber__stripped\": [\"1801\"], \"sfversionnumber\": [1], \"kav_attribute.type\": [\"KB_Article__kav\"], \"kav_validationstatus\": [\"Reviewed\"], \"coveo_foldingparent\": [\"5428021d628a3a6754a3d43ff2322323eb0ca24d8eeae772e728bff1ff18\"], \"collection\": [\"default\"], \"sfcausec\": [\"This node in\\u00a0Coveo.SearchProviger.config<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\n&lt;defaultIndexConfiguration type= ... &gt;</pre>\\n<br>Is missing this children node:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\n&lt;initializeOnAdd&gt;true&lt;/initializeOnAdd&gt;</pre>\\n<br>\\u00a0\"], \"sfsourceid\": [\"50032000015dbB4AAI\"], \"date\": [1517572013], \"kav_lastmodifiedby.id\": [\"00560000003q7JgAAI\"], \"kav_datacategoryselections.datacategorygroupname\": [\"Article_Type;Product\"], \"conversionstate\": [0], \"sfvalidationstatus\": [\"Reviewed\"], \"kav_isdeleted\": [false], \"sflastmodifiedbyid\": [\"00560000003q7JgAAI\"], \"kav_lastmodifiedby.name\": [\"Frederic Cote\"], \"kav_articlenumber\": [\"000001801\"], \"permanentid\": [\"c63324738b8e039143b174f85a0596209ed237cc365a7e4385ce7cf2c8af\"]}",
      "sfbuildc" : "1312",
      "sflastmodifiedbyid" : "00560000003q7JgAAI",
      "syssfownerid" : "00560000003q7JgAAI",
      "sfsitecoreversionc" : "N/A",
      "sourcetype" : "Salesforce",
      "sysconnectortype" : "Salesforce2",
      "sftitle" : "Could not establish trust relationship for SSL/TLS after a app pool recycle",
      "rowid" : 5492054,
      "syssfcreatedbyid" : "00560000003q7JgAAI",
      "sfdescriptionc" : "Using Coveo for Sitecore 3.0 version after March 2016, you get this error message in the diagnostic page<br> \n<pre class=\"ckeditor_codeblock\">\nThe remote server returned an error: (500) Internal Server Error \nCould not establish trust relationship for the SSL/TLS secure channel with authority &#39;myCESServer:52810&#39;</pre>\nIn addition, while this message is visible, no search can be performed<br><br>Also, this message occurs after each App Pool Recycle. Re indexing a single item solves the problem until the next recycle.",
      "size" : 3962,
      "sfislatestversion" : "true",
      "sysdocumenttype" : "article",
      "sfcreateddate" : 1476217361000,
      "sfsourceid" : "50032000015dbB4AAI",
      "clickableuri" : "https://na61.salesforce.com/kA132000000L6sl",
      "syssource" : "Salesforce Public Items",
      "sfownerid" : "00560000003q7JgAAI",
      "orderingid" : 212384990660370855,
      "sfpublishstatus" : "Online",
      "syssize" : 3962,
      "sysdate" : 1517572013000,
      "sflanguage" : "en_US",
      "sysparents" : "<?xml version=\"1.0\" encoding=\"utf-16\"?><parents><parent name=\"Organization\" uri=\"https://na61.salesforce.com/home/home.jsp\" /><parent name=\"KB_Article__ka\" uri=\"http://www.salesforce.com/org:organization/articletype:KB_Article\" /><parent name=\"Could not establish trust relationship for SSL_TLS after a app pool recycle\" uri=\"https://na61.salesforce.com/kA132000000L6sl\" /></parents>",
      "sfdcexpandedarticletype" : "All;All|Problem/Fix",
      "sfcurrencyisocode" : "CAD",
      "sfkbid" : "kA132000000L6slCAC",
      "sfcausec" : "This node in Coveo.SearchProviger.config<br> \n<pre class=\"ckeditor_codeblock\">\n&lt;defaultIndexConfiguration type= ... &gt;</pre>\n<br>Is missing this children node:<br> \n<pre class=\"ckeditor_codeblock\">\n&lt;initializeOnAdd&gt;true&lt;/initializeOnAdd&gt;</pre>\n<br> ",
      "sfdcexpandedproduct" : "All;All|Sitecore",
      "syssfcreateddate" : 1476217361000,
      "sflastmodifieddate" : 1476217823000,
      "sfisvisibleinpkb" : "true",
      "source" : "Salesforce Public Items",
      "sfisdeleted" : "false",
      "sfisvisibleincsp" : "true",
      "sfvalidationstatus" : "Reviewed",
      "sflastmodifiedbyname" : "Frederic Cote",
      "collection" : "default",
      "sfdcproduct" : [ "Sitecore" ],
      "syssourcetype" : "Salesforce",
      "sfarticletype" : "KB_Article__kav",
      "indexeddate" : 1518187508000,
      "connectortype" : "Salesforce2",
      "filetype" : "SalesforceItem",
      "sfcreatedbyname" : "Frederic Cote",
      "sysclickableuri" : "https://na61.salesforce.com/kA132000000L6sl",
      "sysfiletype" : "SalesforceItem",
      "sfvendorversionc" : "Sitecore 7.2.160123",
      "language" : [ "English" ],
      "sffirstpublisheddate" : 1476217823000,
      "sfownername" : "Frederic Cote",
      "sitename" : "Community",
      "sysrowid" : 5492054,
      "uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US",
      "syscollection" : "default"
    },
    "Title" : "Could not establish trust relationship for SSL_TLS after a app pool recycle",
    "Uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US",
    "PrintableUri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US",
    "ClickUri" : "https://na61.salesforce.com/kA132000000L6sl",
    "UniqueId" : "42.13819$http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L6slCAC/language:en_US",
    "Excerpt" : "Using Coveo for Sitecore 3.0 version after March 2016, you get this error message in the diagnostic page ... The remote server returned an error: (500) Internal Server Error ... Could not establish...",
    "FirstSentences" : null
  }